source/components/JsxParser.tsx

```diff
@@ -329,20 +338,32 @@ export default class JsxParser extends React.Component<TProps> {
 			children = children.props.children
 		}
 
-		return React.createElement(component || lowerName, props, children)
+		const componentProps = {
+			...props,
+			lowerName,
+			component,
+			children,
+		}
+
+		return React.createElement(ForwardRefComponent, componentProps)
+		// return React.createElement(component || lowerName, props, children)
 	}
 
-	render = (): JSX.Element => {
+	render() {
 		const jsx = (this.props.jsx || '').trim().replace(/<!DOCTYPE([^>]*)>/g, '')
 		this.ParsedChildren = this.#parseJSX(jsx)
-		const className = [...new Set(['jsx-parser', ...String(this.props.className).split(' ')])]
-			.filter(Boolean)
-			.join(' ')
+		const className = [...new Set(['jsx-parser', ...String(this.props.className)
+			.split(' ')])].filter(Boolean).join(' ')
 
 		return (
 			this.props.renderInWrapper
-				? <div className={className}>{this.ParsedChildren}</div>
-				: <>{this.ParsedChildren}</>
+				? (
+					<div className={className}>{
+						React.createElement(React.Fragment, {}, this.ParsedChildren)
+					}
+					</div>
+				)
```

Comment:
I don't see any benefit to this change. It introduces a fragment where one isn't needed, and uses create element where it also isn't needed.